src/app/Components/event-inspector/event-inspector.component.ts

```diff
@@ -7,9 +8,13 @@ import { Component, OnInit } from '@angular/core';
 })
 export class EventInspectorComponent implements OnInit {
 
-  constructor() { }
+  eventMessage: any;
```

Comment:
Using type 'any' is not really recommended. Type 'string' would be a more appropriate choice here.